SimplePolicy/SimplePolicyPlugin.cs

```diff
@@ -60,13 +60,13 @@ private static IEnumerable<Transaction> FilterForBlock_Policy2(IEnumerable<Trans
             if (!(transactions is IReadOnlyList<Transaction> tx_list))
                 tx_list = transactions.ToArray();
 
-            Transaction[] free = tx_list.Where(p => p.IsLowPriority)
+            Transaction[] free = tx_list.Where(p => IsLowPriority(p))
                 .OrderByDescending(p => p.NetworkFee / p.Size)
                 .ThenByDescending(p => p.NetworkFee)
                 .Take(Settings.Default.MaxFreeTransactionsPerBlock)
                 .ToArray();
 
-            Transaction[] non_free = tx_list.Where(p => !p.IsLowPriority)
+            Transaction[] non_free = tx_list.Where(p => !IsLowPriority(p))
```

Comment:
Ok, ClaimTx will enter here because its not LowPriority, correct? This is the part Im trying to.test (but will only be able to finish tomorrow). If ClaimTx netfee is always zero, wouldnt it be sent forever to the end of the queue?
I guess Peter mentioned this situation... so if thats the case, we will need to insert it together with the free tx (20 maximum), with bigger priority than all other free tx, and not subject to maxfreetxsizelimit. Please confirm if Im mistaken here.